internal/repos/github.go

```diff
@@ -742,10 +755,13 @@ func (q *repositoryQuery) Do(ctx context.Context, results chan *githubResult) {
 			return
 		}
 
-		switch {
-		case res.TotalCount > q.Limit:
+		if res.TotalCount > q.Limit {
 			log15.Info(
-				fmt.Sprintf("repositoryQuery matched more than %d results, refining it and retrying", q.Limit),
+				"repositoryQuery matched more than limit, refining",
+				"limit",
+				q.Limit,
+				"results count",
```

Comment:
nit: we normally treat keys like programming keywords (no spaces)
```suggestion
				"count",
```